src/controller/eme-controller.js

```diff
@@ -4,6 +4,8 @@
  * DRM support for Hls.js
  */
 
+/* eslint-disable indent */
```

Comment:
Why do we need this?

Reply:
related to
https://github.com/Tubitv/hls.js/pull/12/files#diff-1cb8b36ed42a658ab63ac970ab1bd648d67f9b199f95fd279f8a50c8d51df2a6R343-R357

I'm not sure why it'll report `indent` error for this `switch` block.